### PR TITLE
Static Home page UI

### PR DESCRIPTION
**Pull request by dikshadhote**
Create note UI
image

PS: Please bear with me, code formatter not working

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -12,6 +13,7 @@
 
 body {
   background-color: var(--black-dark-background);
+  color: #f1e5e5a6;
```
Comment (t007rushi) [inline src/App.css]: use variable here also

File: src/components/Home/Home.js [hunk 2]
```diff
@@ -1,10 +1,49 @@
...
+      <div className="d-flex flex-justify-center">
+        <div className="create-note-container p-1">
+          <div className="d-flex flex-justify-space-between align-items-center">
+            <input
+              className="input-create-note white-text-color fs-2"
+              placeholder="Title"
+            />
+            <BsPinAngleFill
+              className="white-text-color cursor-pointer"
+              title="Pin this note"
```
Comment (t007rushi) [inline src/components/Home/Home.js:19]: nice that used title attribute

**Review by t007rushi — APPROVED** [review]
Host it for easy preview

File: src/App.css (hunk 1 above)
Comment (dikshadhote) [reply src/App.css]: Its not needed, removed it

File: src/components/Home/Home.js (hunk 2 above)
Comment (dikshadhote) [reply src/components/Home/Home.js:19]: Thanks

**Review by rushikesh3299 — APPROVED** [review]
Looks good to me. Liked the way you used utility classes.

**Comment by dikshadhote** [comment]
> Looks good to me. Liked the way you used utility classes.

**Review by rkapoor10 — COMMENTED** [review]
- The code is well structured
 - The UI is great
 - very well written css
 - `input:focus-visible {
  outline: none;
}` will use this

**Pull request merged**